### DISCUSSION TITLE
How to startup

### DISCUSSION THREAD

**Discussion by alvin2929** [iptv-org/community#109] [Q&A] [answered]
Hi i have alliptvplayer on my lg. i tried couple url and its not working. Can someone walk me through how to do it? I had duplex play before was so east but now OMG. So complicated

**Comment by benevenstanciano** [comment]
Hi,
It's likely that you're using the URL for viewing/editing the code, rather than the URL which actually points to the playlist, if that makes sense. If you open the playlist you want to use, you can see in the top right a button that says "Raw". It's this link that you want to use. For example, if you want the UK playlist, the URL you'd put into the player would be: https://raw.githubusercontent.com/iptv-org/iptv/master/streams/uk.m3u
Sometimes different players can't handle different types of link so if that doesn't work then maybe there's other options.

**Comment by freearhey** [comment, marked as answer]
Try using this link:

```
https://iptv-org.github.io/iptv/index.m3u
```
